src/Raven.Server/Config/Categories/LogsConfiguration.cs

```diff
@@ -9,58 +10,78 @@ namespace Raven.Server.Config.Categories
     [ConfigurationCategory(ConfigurationCategoryType.Logs)]
     public sealed class LogsConfiguration : ConfigurationCategory
     {
-        [Description("The path to the directory where the RavenDB server logs will be stored")]
+        [DefaultValue(null)]
+        [ConfigurationEntry("Logs.ConfigPath", ConfigurationEntryScope.ServerWideOnly)]
+        public PathSetting ConfigPath { get; set; }
+
         [DefaultValue("Logs")]
         [ConfigurationEntry("Logs.Path", ConfigurationEntryScope.ServerWideOnly)]
         public PathSetting Path { get; set; }
 
-        [Description("The level of logs that will be written to the log files (None, Operations or Information)")]
-        [DefaultValue(LogMode.Operations)]
-        [ConfigurationEntry("Logs.Mode", ConfigurationEntryScope.ServerWideOnly)]
-        public LogMode Mode { get; set; }
+        [DefaultValue(LogLevel.Info)]
+        [ConfigurationEntry("Logs.MinLevel", ConfigurationEntryScope.ServerWideOnly)]
+        public LogLevel MinLevel { get; set; }
 
-        [Description("Determine whether logs are timestamped in UTC or with server-local time")]
-        [DefaultValue(true)]
-        [ConfigurationEntry("Logs.UseUtcTime", ConfigurationEntryScope.ServerWideOnly)]
-        public bool UseUtcTime { get; set; }
+        [DefaultValue(null)]
+        [ConfigurationEntry("Logs.NLog.Path", ConfigurationEntryScope.ServerWideOnly)]
+        public PathSetting NLogInternalPath { get; set; }
+
+        [DefaultValue(LogLevel.Info)]
+        [ConfigurationEntry("Logs.NLog.Level", ConfigurationEntryScope.ServerWideOnly)]
+        public LogLevel NLogInternalLevel { get; set; }
+
+        [DefaultValue(false)]
+        [ConfigurationEntry("Logs.NLog.LogToConsole", ConfigurationEntryScope.ServerWideOnly)]
+        public bool NLogLogToConsole { get; set; }
+
+        [DefaultValue(false)]
+        [ConfigurationEntry("Logs.NLog.LogToConsoleError", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
Why do we have NLog specific configurations? Why not dropping `.NLog` in the config option names?